tests/content/input/21.formats props with multiple child props.md

```diff
@@ -0,0 +1,20 @@
+::page-section
```

Comment:
This covers the issue scenario, whereby when running the formatter, the `alt-txt` property (the second child prop) was incorrectly being "unindented" to the same level as the parent `image:` property.